src/sandbox.rkt

```diff
@@ -106,8 +107,19 @@
               ,oracle-errs ,all-alts)
        (match-define (list newpoints newexacts) (get-p&es newcontext))
        (match-define (list points exacts) (get-p&es context))
-       (test-success test bits (- (current-inexact-milliseconds) start-time) timeline
-                     start end points exacts
+       (define start-prog (alt-program start))
+       (define end-prog (alt-program end))
+       (define start-resugared (make-alt
```

Comment:
Yo, you can't do it this way, the `make-alt` drops the whole derivation chain that the old alt stored. You need to create a new alt with the old one as its parent and some kind of event like `resugar` that you add in `web/make-graph.rkt`. Alternatively maybe we should only resugar programs when we show them, not in the overall `test-success` record.